apps/governance/src/routes/rewards/epoch-individual-rewards/epoch-individual-rewards.tsx

```diff
@@ -99,17 +100,24 @@ export const EpochIndividualRewards = ({
     prevEpochIdRef.current = epochId;
   }, [epochId, refetchData]);
 
+  // Workarounds for the error handling of AsyncRenderer
+  const filteredErrors = filterExpectedErrors(error);
+  const filteredData = data || [];
+
   return (
     <AsyncRenderer
       loading={loading}
-      error={error}
-      data={data}
+      error={filteredErrors}
```

Comment:
the same issue I've seen when I was trying to cast a vote but there were no associated tokens on my account. 
could you please check `usePoolForDelegations` where `isPartyNotFoundError` gets triggered?

Reply:
Done